test/e2e/util.go

```diff
@@ -108,7 +108,7 @@ func waitSizeReachedWithFilter(t *testing.T, f *framework.Framework, clusterName
 				names = append(names, pod.Name)
 			}
 		}
-		t.Logf("waiting size (%d), etcd pods: %v", size, names)
+		t.Logf("%v waiting size (%d), etcd pods: %v", time.Now(), size, names)
```

Comment:
I was having some error "not enough arguments.
But I am able to make it work like this:
```
t.Log(time.Now(), fmt.Sprintf(format, args...))
```